crates/ruff_linter/src/rules/pycodestyle/rules/invalid_escape_sequence.rs

```diff
@@ -28,20 +28,38 @@ use crate::fix::edits::pad_start;
 /// ## References
 /// - [Python documentation: String and Bytes literals](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals)
 #[violation]
-pub struct InvalidEscapeSequence(char);
+pub struct InvalidEscapeSequence {
+    ch: char,
+    fix_title: FixTitle,
+}
 
 impl AlwaysFixableViolation for InvalidEscapeSequence {
     #[derive_message_formats]
     fn message(&self) -> String {
-        let InvalidEscapeSequence(char) = self;
-        format!("Invalid escape sequence: `\\{char}`")
+        let InvalidEscapeSequence { ch, .. } = self;
+        format!("Invalid escape sequence: `\\{ch}`")
     }
 
     fn fix_title(&self) -> String {
-        "Add backslash to escape sequence".to_string()
+        match self.fix_title {
+            FixTitle::AddBackslash => format!("Add backslash to escape sequence"),
```

Comment:
Should we add one of this fix to the examples?

Reply:
Yeah, I've added it. Thanks